editor/block-mover/index.js

```diff
@@ -15,7 +15,8 @@ import { IconButton } from 'components';
 import './style.scss';
 import { isFirstBlock, isLastBlock } from '../selectors';
 
-function BlockMover( { onMoveUp, onMoveDown, isFirst, isLast } ) {
+// Only exported for testing.
```

Comment:
I will stop adding this comment in for connected components or similar things.  Is the runtime code not stripping comments out?